CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-oauth2-client ChangeLog
 
+## 4.0.0 - 2021-xx-xx
+
+### Removed
+- **BREAKING**: Removed database storage.
```

Comment:
At the end of the day, were there other breaking changes for public APIs?  If so, please document.

Reply:
Updated.